src/utils/windowDimensions.js

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from 'react';
+
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
```

Comment:
Wow! This is great!  I never would have even thought to do this to make the apge more responsive to dimension changes